src/content/changelogs/ai-gateway.yaml

```
@@ -5,10 +5,31 @@ productLink: "/ai-gateway/"
 productArea: Developer platform
 productAreaLink: /workers/platform/changelog/platform/
 entries:
+  - publish_date: "2024-09-26"
+    title: Persistent logs
+    description: |-
+      * **Logs**: AI Gateway now has [logs that persist](/ai-gateway/observability/logging/index.mdx), giving you the flexibility to store them for your preferred duration.
```

```suggestion
      * **Logs**: AI Gateway now has [logs that persist](/ai-gateway/observability/logging/index), giving you the flexibility to store them for your preferred duration.
```

Not sure if you need the `.mdx`.